worlds/tunic/locations.py

```diff
@@ -325,6 +325,10 @@ class TunicLocationData(NamedTuple):
 
 location_name_groups: Dict[str, Set[str]] = {}
 for loc_name, loc_data in location_table.items():
+    loc_group_name = loc_name.split(" - ", 1)[0]
+    if loc_group_name not in location_name_groups.keys():
+        location_name_groups[loc_group_name] = set()
+    location_name_groups[loc_group_name].add(loc_name)
```

Comment:
```
location_name_groups: Dict[str, Set[str]] = {}
for loc_name, loc_data in location_table.items():
    loc_group_name = loc_name.split(" - ", 1)[0]
    location_name_groups.setdefault(loc_group_name, set()).add(loc_name)
```

or, in my opinion even better

```
location_name_groups: Dict[str, Set[str]] = defaultdict(lambda: set())
for loc_name, loc_data in location_table.items():
    loc_group_name = loc_name.split(" - ", 1)[0]
    location_name_groups[loc_group_name).add(loc_name)
```

Reply:
Revised it to the first suggestion cause I understand that one better